content/rest/channels-messages.textile

```diff
@@ -209,6 +224,12 @@ if result_page.has_next?
   puts "#{next_page.items.length} messages received in second page"
 end
 
+bc[python]. result_page = channel.history()
+print str(len(result_page.items)) + " messages received in first page"
```

Comment:
No good reason, nope. will change
(my Elixir brain is used to typing double quotes for strings, single quotes in Elixir are a different datatype)